gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -75,14 +75,16 @@
 public class MapContainerViewModel extends AbstractViewModel {
 
   // Higher zoom levels means the map is more zoomed in. 0.0f is fully zoomed out.
-  private static final float DEFAULT_FEATURE_ZOOM_LEVEL = 18.0f;
+  public static final float ZOOM_LEVEL_THRESHOLD = 16f;
+  public static final float DEFAULT_FEATURE_ZOOM_LEVEL = 18.0f;
   private static final float DEFAULT_MAP_ZOOM_LEVEL = 0.0f;
   private static final Point DEFAULT_MAP_POINT =
       Point.newBuilder().setLatitude(0.0).setLongitude(0.0).build();
   private final LiveData<Loadable<Project>> projectLoadingState;
   private final LiveData<ImmutableSet<MapFeature>> mapFeatures;
   private final LiveData<BooleanOrError> locationLockState;
   private final LiveData<Event<CameraUpdate>> cameraUpdateRequests;
+  private final LiveData<Event<Nil>> zoomThresholdUpdateRequests;
```

Comment:
Is replay required here? In other words, is there a chance the event could happen before subscribe/observe? If not, we can (should?) expose this as an `Observable<>` instead of `LiveData<Event>`. My impression is that `LiveData` is better suited for representing reactive state rather than events.

Reply:
Good point, I don't think this event could happen before subscribe/observe so I think we can use `Observable<>`.

I've updated to `Observable<>` and verified the behavior in app.